src/mantarray_file_manager/files.py

```diff
@@ -410,6 +411,13 @@ def get_raw_tissue_reading(self) -> NDArray[(2, Any), int]:
             times = np.arange(len(tissue_data), dtype=np.int32) * time_step
             len_time = len(times)
 
+            new_time_delta = self._check_for_trimmed_file(
+                times, time_delta_centimilliseconds
+            )
```

Comment:
note for future: if you are already passing in `time_delta_centimilliseconds` into that method, then you can just have the method return that right back if there was no new time delta, and say `time_delta_centimillisceonds = self._check_for...` and avoid the follow-up `if` statement